### DISCUSSION TITLE
Is the full documentation inactive?

### DISCUSSION THREAD

**Discussion by je4npw** [noxoua/filament-activity-log#83] [Q&A] [answered]
I recently tried to access the documentation at the urlk activity-log.noxo.app/ but it appears to be unavailable. 
Is there another way to access the docs?

Thank you for the nice job!
![image](https://github.com/user-attachments/assets/87e176a3-10bd-4514-8934-7ebf3e5da3e3)
![image](https://github.com/user-attachments/assets/a15755b0-f8d7-4708-a19d-ef12fdd4275f)

**Comment by noxoua** [comment, marked as answer]
updated - https://noxoua.github.io/filament-activity-log/

**Reply by je4npw** [reply]
Thanks a lot, man! Sending big hugs all the way from Brazil! 🇧🇷
